app/views/taxon_ranges/show.html.haml

```diff
@@ -0,0 +1,52 @@
+- content_for :title do
+  Taxon Range for
+  = render "taxa/taxon.txt.erb", taxon: @taxon_range.taxon
+
+.container
+  .row
+    .col-xs-12
+      %h2.pull-left
+        =t :taxon_range_for_taxon_html, taxon: render( "shared/taxon", taxon: @taxon_range.taxon, named_ancestor: true, link_url: @taxon_range.taxon )
+      - if current_user.is_curator?
+        .pull-right= link_to t( :edit ), edit_taxon_range_path( @taxon_range )
+  .row.cf#top
+    .col-sm-3.col-xs-12.pull-right
+      .panel.panel-info
+        .panel-heading
+          %h3.panel-title
+            =t :about_taxon_ranges
+        .panel-body
+          %p
+            =raw t :about_taxon_ranges_desc, url: "/pages/taxon_ranges"
+          %img.media-object{ src: "https://static.inaturalist.org/wiki_page_attachments/2817-original.png", class: "img-responsive" }
```

Comment:
I don't think this needs to block merge, but this is a bad habit: images that the app uses should be checked into the repo and served via the asset pipeline.